pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -570,6 +613,124 @@ def extract_ifos_and_vetoes(trig_file, veto_files, veto_cat):
 # =============================================================================
 # Function to load injections
 # =============================================================================
+def load_missed_found_injections(hdf_file, ifos, opts,
```

Comment:
I have the same comment here as in the one below, I'm not sure is that maintaing-oriented in the long run

Reply:
Ok, I see how having this function helps you keep `pycbc_pygrb_page_tables` tidy.  Since this function will be extracting the template parameters too and thus be tailored to `pycbc_pygrb_page_tables`, I would define it in `pycbc_pygrb_page_tables` rather than in `pycbc/results/pygrb_postprocessing_utils.py`